### PR TITLE
* Implement #866

### PR DESCRIPTION
**Pull request by Wagnerp**
* Implements #866

image

kmb866

### CODE REVIEW

**Comment by Smurf-IV** [comment]
Comments left.

**Comment by Smurf-IV** [comment]
In the demo "Application" does not show the Krypton Icon of the parent form ??

**Comment by Wagnerp** [comment]
> Comments left.

@Smurf-IV Which file?

**Comment by Wagnerp** [comment]
> In the demo "Application" does not show the Krypton Icon of the parent form ??

Need to add the parameter

**Comment by Smurf-IV** [comment]
image

**Comment by Wagnerp** [comment]
> image

@Smurf-IV Could you make the image bigger please, as the text is too small to read? :)

**Comment by Smurf-IV** [comment]
Just click on the link "View in Github" in the email, and it takes you to the page:
https://github.com/Krypton-Suite/Standard-Toolkit/pull/871#issuecomment-1356122606

**Comment by Wagnerp** [comment]
> Just click on the link "View in Github" in the email, and it takes you to the page: [#871 (comment)](https://github.com/Krypton-Suite/Standard-Toolkit/pull/871#issuecomment-1356122606)

Tried, but it just takes me to this thread. In other repos, I can normally find the comments, not so with this one?

**Comment by Wagnerp** [comment]
image

**Comment by Smurf-IV** [comment]
That looks right, scroll back up the page....
Or use another browser (I use FireFox for this site), as it might be due to a blocked Github script.

**Comment by Wagnerp** [comment]
> That looks right, scroll back up the page.... Or use another browser (I use FireFox for this site), as it might be due to a blocked Github script.

This is what I see...

image

Could it be Edge (Chromium)?

**Comment by Wagnerp** [comment]
Not working in Firefox for me either...

**Comment by Smurf-IV** [comment]
I get your view in Edge.
Until I then login that I get the views I see.

So it seems you need to login to see the Reviews comments and not just the conversation comments left.

**Comment by Wagnerp** [comment]
> I get your view in Edge. Until I then login that I get the views I see.
> 
> So it seems you need to login to see the Reviews comments and not just the conversation comments left.

Odd, I think I can read the comments if I squint at it

**Comment by Wagnerp** [comment]
Addressed feedback

image

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Components/Krypton.Toolkit/Utilities/GraphicsExtensions.cs [hunk 1]
```diff
@@ -128,30 +128,52 @@ public enum IconType
         public static Image SetIcon(Image image, Size size) => (Image)new Bitmap(image, size);
     }
 
+    /// <summary>Specifies the icon type for <see cref="KryptonMessageBox"/>.</summary>
+    [Flags]
```
[the remark this answers is not in the document]
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/GraphicsExtensions.cs]: This just popped???

[the remark this answers is not in the document]
Comment (Smurf-IV) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/GraphicsExtensions.cs]: Just need to sort / resolve the other comments now :-)

File: Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonMessageBoxForm.cs [hunk 2]
```diff
@@ -162,15 +169,27 @@ private void UpdateIcon()
                     {
                         _messageIcon.Image = MessageBoxResources.Asterisk;
                     }
-                    SystemSounds.Asterisk.Play();
+                    PlayAsteriskSound();
```
[the remark this answers is not in the document]
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Visuals/KryptonMessageBoxForm.cs]: Resolved

File: Source/Krypton Components/Krypton.Toolkit/Utilities/AssemblyUtilities.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
+﻿#region BSD License
+/*
+ * 
+ * Original BSD 3-Clause License (https://github.com/ComponentFactory/Krypton/blob/master/LICENSE)
```
[the remark this answers is not in the document]
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/AssemblyUtilities.cs]: Deleted old statement

File: Source/Krypton Components/Krypton.Toolkit/Utilities/KryptonMessageBoxIcon.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
+﻿namespace Krypton.Toolkit;
+
+/// <summary>Specifies the icon type for <see cref="KryptonMessageBox"/>.</summary>
+[Flags]
```
[the remark this answers is not in the document]
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Utilities/KryptonMessageBoxIcon.cs]: Removed `[Flags]`

**Pull request merged**